Formula/glslviewer.rb

```diff
@@ -1,8 +1,9 @@
 class Glslviewer < Formula
   desc "Live-coding console tool that renders GLSL Shaders"
   homepage "http://patriciogonzalezvivo.com/2015/glslViewer/"
-  url "https://github.com/patriciogonzalezvivo/glslViewer/archive/1.6.8.tar.gz"
-  sha256 "3792984457c9487cd1f87374b7698f3b66f4480b3eeac7d940a2ecf9e74dee3e"
+  url "https://github.com/patriciogonzalezvivo/glslViewer.git",
+    tag:      "1.7.0",
+    revision: "d698ddbff0dc119b506334494895b546ae9e4ae8"
```

Comment:
Why do we need a Git clone?

Reply:
There's a `miniaudio` submodule and the Makefile runs a `git` command to update it. We could probably make the submodule a separate formula and prevent the `git` command from being run instead.

Reply:
One thing to note is that the submodule (https://github.com/mackron/miniaudio) doesn't actually tag their releases (e.g. mackron/miniaudio#273).

Reply:
Specific commits have versions, though, which is good enough for me. (It'll be a pain to `livecheck` though, I think.) A separate `miniaudio` formula sounds good.

Reply:
Is a formula better or a `resource` block since it is just a single header file that needs to be located in a specific location `include/miniaudio/miniaudio.h`

Reply:
If it doesn't have tagged releases it should be a resource block.